pyproject.toml

```diff
@@ -65,6 +65,16 @@ homepage = "https://github.com/NREL/sup3r"
 documentation = "https://nrel.github.io/sup3r/"
 repository = "https://github.com/NREL/sup3r"
 
+[tool.ruff]
```

Comment:
good add :). i can send you my list of codes that seem to best match flake8 + pylint

Reply:
Yes, please! But I would add those in a different PR so we don't mix the potential source of errors.